Yoast/wordpress-seo
# yoastseo refactor integration

## Context


*

## Summary


This PR can be summarized in the following changelog entry:

* Implements the necessary steps to integrate the new `yoastseo` structure with the plugin.

## Relevant technical choices:

* Makes individual language researcher files available on window.
* Expands plugin webpack config to build language files.
* Load language-specific researcher/enqueue language scripts in
class-admin-asset-manager.php.
* In all places where localized scripts for the analysis are enqueued we also enqueue the currently active language.


## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

The steps below serve to smoke-test the plugin integration of the refactor functionality.

Test a post in 3 languages:
- English
- A supported language
- A language that we don't have specific support for

Test the following 3 environments (testing each in one language is sufficient):
- A normal post
- Taxonomy page
- A normal post in Elementor

Steps to test:

1.  Confirm that the SEO and readability analysis load
2. Set a keyphrase, add it to the text and see that it's recognized
3. Change some readability metrics (e.g., add transition words) and confirm that the analysis updates accordingly

### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

*

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes #
